.github/workflows/shiftleft.yml

```diff
@@ -0,0 +1,54 @@
+---
+# This workflow integrates ShiftLeft NG SAST with GitHub
+# Visit https://docs.shiftleft.io for help
+name: ShiftLeft
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  NextGen-Static-Analysis:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Download ShiftLeft CLI
+      run: |
+        curl https://cdn.shiftleft.io/download/sl > ${GITHUB_WORKSPACE}/sl && chmod a+rx ${GITHUB_WORKSPACE}/sl
```

Comment:
`curl | bash` ?    Isn't there a way to not download it every time?

Reply:
My error. I did not realize that the pull request picked up the ShiftLeft files. I'll resubmit separately.